### ISSUE TITLE
mbtileserver and nginx unit 

### ISSUE THREAD

**Issue by catalinconstant** [consbio/mbtileserver#97]
It is possible to have an mbtileserver version o  that will work with Nginx UNIT ?
https://unit.nginx.org/howto/samples/#sample-go
Thank you

**Comment by brendan-ward** [comment]
We'd have to switch to using UNIT as the web framework instead of `echo`.   I don't see the advantage after reading through the UNIT docs; is there something obvious I'm missing?

**Comment by catalinconstant** [comment]
Currently I am using tileserver-gl which use express framework,  it was very easy to make it work with nginx UNIT just few lines of code
<code>
const {
  createServer,
  IncomingMessage,
  ServerResponse,
} = require('unit-http')
require('http').ServerResponse = ServerResponse
require('http').IncomingMessage = IncomingMessage
const express = require('express')
const app = express()
</code>
and few more tweaks.
I was thinking that this is possible with echo framework in GO without changing the application just the http handling through unit-http.
In my oppinion UNIT clear has some advantage as an application server orchestarting different microservices from a dingle point.
Since I am quite new to GO I did not know the implications for mbtileserver.
Anyway I started testing mbtileserver and I am quite impresed by its performance.
